### PR TITLE
3.6.3: Corrections fr-FR and fr-CA installation ini strings

### PR DESCRIPTION
**Pull request by infograf768**
fr-FR and fr-CA got a typo and for fr-CA a change was necessary from e-mail to courriel.

Thanks for merging before release

### CODE REVIEW

**Comment by infograf768** [comment]
Simple changes. RTC for 3.6.3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12370">issues.joomla.org/tracker/joomla-cms/12370</a>.</sub>

File: installation/language/fr-CA/fr-CA.ini [hunk 1]
```diff
@@ -339,16 +339,3 @@ JLIB_JS_AJAX_ERROR_NO_CONTENT="Aucun contenu obtenu."
 JLIB_JS_AJAX_ERROR_OTHER="Une erreur s'est produite lors de la récupération des données JSON: HTTP %s status code."
 JLIB_JS_AJAX_ERROR_PARSE="Une erreur de parsing s'est produite lors de la récupération des données JSON suivantes :<br/><code style="_QQ_"color:inherit;white-space:pre-wrap;padding:0;margin:0;border:0;background:inherit;"_QQ_">%s</code>"
 JLIB_JS_AJAX_ERROR_TIMEOUT="Un délai d’attente (timeout) s'est produit lors de la récupération des données JSON."
-
-[French specific]
```
Comment (wilsonge) [inline installation/language/fr-CA/fr-CA.ini]: Do we not need these strings?

Comment (infograf768) [inline installation/language/fr-CA/fr-CA.ini]: They are needed by fr-FR in their community distros. It has been considered a good idea to let them in core to not get mixed up. 
Therefore they are not for fr-CA for which there are no distros.

Comment (wilsonge) [inline installation/language/fr-CA/fr-CA.ini]: :+1:

**Pull request merged**